docs/kafka/service-binding-kafka/README.adoc

```diff
@@ -752,3 +755,5 @@ You should now see that the producer has connected to the Kafka instance. The pr
 . Open the logs for the pod of the consumer component, in the same way that you did for the producer component.
 +
 You should now see that the consumer has connected to the Kafka instance. The consumer displays the same country names that the producer sends to the `countries` Kafka topic, and in the same order.
++
```

Comment:
Should we let people know at the end or begining ?

Reply:
It's a good question, @wtrocki. But I think we're putting this information in the appropriate place. It's not until the end of each example that users actually have messages to see.

Reply:
Yes. Although produce UI will be also added in upcomming sprints.